### PR TITLE
Weekly dependency updates

### PR DESCRIPTION
**Pull request by jmartin-tech**
This PR should have been auto-generated by [actions/github-script](https://github.com/actions/github-script). `bundle update` revealed the following gems have new version to be evaluated for update.

More adjustment to get automation are in progress, for now let us pick this week.

## Verification

List the steps needed to make sure this thing works

- [x] Security review diffs for updated gems

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -445,7 +445,7 @@ GEM
     ruby-progressbar (1.11.0)
     ruby-rc4 (0.1.5)
     ruby2_keywords (0.0.5)
-    ruby_smb (3.1.1)
+    ruby_smb (3.1.2)
```
Comment (adfoster-r7) [inline Gemfile.lock:448]: cc @zeroSteiner

Is this bump blocked on https://github.com/rapid7/metasploit-framework/pull/16481 - or can this go out independently 👀

Comment (zeroSteiner) [reply Gemfile.lock:448]: That should be fine to go out. There are no breaking changes that I'm aware of.

It includes:
* rapid7/ruby_smb#213
* rapid7/ruby_smb#214
* rapid7/ruby_smb#215
* rapid7/ruby_smb#216
* rapid7/ruby_smb#217

File: Gemfile.lock [hunk 2]
```diff
@@ -394,7 +394,7 @@ GEM
       metasm
       rex-core
       rex-text
-    rex-socket (0.1.35)
+    rex-socket (0.1.38)
```
[the remark this answers is not in the document]
Comment (gwillcox-r7) [reply Gemfile.lock:397]: Version 0.1.40 should fix the issue raised above so will update this to bump to that version.

[the remark this answers is not in the document]
Comment (gwillcox-r7) [reply Gemfile.lock]: Bumped version this should now be fixed.

**Pull request merged**